### PR TITLE
fix margin-top bug in footer

### PR DESCRIPTION
**Pull request by MahendraBishnoi29**
## Description
fix the extra margin-top bug in the footer

## Related Issue
#9770

### CODE REVIEW

**Comment by MahendraBishnoi29** [comment]
before
[4b307656-b3c7-4bb8-a136-3da981ec9498.webm](https://user-images.githubusercontent.com/74294202/226942069-383f5bcd-0a06-4fd4-b23d-98a65c69c732.webm)

After
Screenshot 2023-03-22 201627

File: src/pages-conditional/dapps.tsx
Comment (MahendraBishnoi29) [inline src/pages-conditional/dapps.tsx:1]: @corwintines @pettinarip please undo this commit. this was made accidentally, this should have a separate PR for this issue #9772

Comment (MahendraBishnoi29) [reply src/pages-conditional/dapps.tsx:1]: or should I close this PR and make 2 new PRs for both of them? @corwintines @pettinarip

**Comment by corwintines** [comment]
Thanks @MahendraBishnoi29. Ill close this so that you can make 2 separate PRs. Makes reviewing based on the issues a bit easier.

Also, for the second portion here, I would recommend removing and not just commenting out when you submit the separate PRs. Thanks!

**Comment by MahendraBishnoi29** [comment]
> Thanks @MahendraBishnoi29. Ill close this so that you can make 2 separate PRs. Makes reviewing based on the issues a bit easier.
> 
> Also, for the second portion here, I would recommend removing and not just commenting out when you submit the separate PRs. Thanks!

thanks @corwintines submitting the PRs right now 🚀

**Pull request closed without merging**